### DISCUSSION TITLE
Suboptimal link for workflow in `[GitHub] The ":workflow" workflow in :repo will be disabled soon`

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#14221] [Actions] [answered]
<img width="531" alt="image" src="https://user-images.githubusercontent.com/2119212/161463140-3a8e6818-7034-49ab-ab44-a49927ddf350.png">

This link:
[Manage workflow](https://github.com/jsoref/homebrew-core/actions?query=workflow%3A%22Remove+disabled+formulae%22)

Leads to:
<img width="1121" alt="image" src="https://user-images.githubusercontent.com/2119212/161463291-b5059205-b500-4d2f-9c81-81c6c7e4e409.png">

Which is not as helpful as:
https://github.com/jsoref/homebrew-core/actions/workflows/remove-disabled-formulae.yml?query=workflow%3A%22Remove+disabled+formulae%22
<img width="1116" alt="image" src="https://user-images.githubusercontent.com/2119212/161463341-5a671856-1b75-4cd6-877e-03b961f6c2bb.png">

Which would let users enable/disable the workflow.

**Comment by ethomson** [comment, marked as answer]
Thanks @jsoref - this is good feedback indeed. I'll put this on the team's backlog!
